bigchaindb/fastquery.py

```diff
@@ -1,60 +0,0 @@
-from bigchaindb.utils import condition_details_has_owner
-from bigchaindb.backend import query
-from bigchaindb.common.transaction import TransactionLink
-
-
-class FastQuery:
```

Comment:
This class doesn't have much to do with votes but If it is being removed in this PR it would be better to remove it from the tests too i.e. `tests/test_fastquery.py` and remove its mocking from `tests/db/test_bigchain_api.py`

Reply:
I think it's better to do it in a separate PR.
I'll remove it and also take care of the new PR